src/main/webapp/authentication-app/yarn.lock

```diff
@@ -0,0 +1,10874 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Same thing as the package-lock.json. Maybe ask @NathanBricault or @Snehalb26 about whether these are necessary to commit since they are such large files